qiskit/ignis/experiments/calibration/methods/cross_resonance.py

```diff
@@ -21,8 +22,9 @@
 from qiskit.ignis.experiments.calibration.instruction_data import InstructionsDefinition
 from qiskit.ignis.experiments.calibration.cal_base_experiment import BaseCalibrationExperiment
 from qiskit.ignis.experiments.calibration.analysis.trigonometric import CosinusoidalFit
+from qiskit.ignis.experiments.calibration.analysis.fit_utils import get_period_fraction
 
-from qiskit.pulse import ControlChannel
+from qiskit.pulse import ControlChannel, Play
 
 
 class RoughCRAmplitude(BaseCalibrationExperiment):
```

Comment:
I think this class can also take `update_pulses` defaults to `'cr': 0.5`. User may be interested in calibrating different angle CR.

Reply:
done in 5b4eaa9